CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ set(DIR_3RDPARTY ${FileIO_SOURCE_DIR}/3rdparty)
 MESSAGE(" PROJECT_SRC = : ${PROJECT_SRC}")
 include_directories(${PROJECT_SRC})
 SET(ACTIVE_CPP0xx_DIR "Release")
-find_library(TCMALLOC tcmalloc PATHS /usr/local/probe/lib )
```

Comment:
@KjellKod, I don't know of a use case where this will matter. I think any use of FileIO will be temporary and performance would not be improved with tcmalloc. You could check pcap file reconstruction against this change. There's a heavy use of FileIO in that code.